<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>466</width>
    <height>554</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>#9Slide03 Arima Madurai ExtraBo</family>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>211</width>
      <height>201</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>#9Slide03 Arima Madurai ExtraBo</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background: rgb(244, 238, 255);
color: rgb(66, 72, 116);</string>
    </property>
    <property name="title">
     <string>Nhập dữ liệu:</string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QTextEdit" name="textEditNhap">
       <property name="font">
        <font>
         <family>#9Slide03 Arima Madurai ExtraBo</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background: rgb(220, 214, 247);</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>20</y>
      <width>351</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>#9Slide03 Arima Madurai ExtraBo</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(66, 72, 116);</string>
    </property>
    <property name="text">
     <string>SỬ DỤNG CÁC HÀM XỬ LÝ CHUỖI</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonNhap">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>280</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>#9Slide03 Arima Madurai ExtraBo</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background: rgb(220, 214, 247);</string>
    </property>
    <property name="text">
     <string>Nhập dữ liệu</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>70</y>
      <width>211</width>
      <height>201</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>#9Slide03 Arima Madurai ExtraBo</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background: rgb(244, 238, 255);
color: rgb(66, 72, 116);</string>
    </property>
    <property name="title">
     <string>Kết quả:</string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QTextEdit" name="textEditKetQua">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="font">
        <font>
         <family>#9Slide03 Arima Madurai ExtraBo</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background: rgb(220, 214, 247);</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="pushButtonDemKTHoa">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>280</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>#9Slide03 Arima Madurai ExtraBo</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background: rgb(220, 214, 247);</string>
    </property>
    <property name="text">
     <string>Đếm số ký tự hoa</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonInSoTuTrenMoiDong">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>320</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>#9Slide03 Arima Madurai ExtraBo</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background: rgb(220, 214, 247);</string>
    </property>
    <property name="text">
     <string>In số từ trên mỗi dòng</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonInChuHoa">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>320</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>#9Slide03 Arima Madurai ExtraBo</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background: rgb(220, 214, 247);</string>
    </property>
    <property name="text">
     <string>In chữ hoa</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonDemSoTu">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>360</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>#9Slide03 Arima Madurai ExtraBo</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background: rgb(220, 214, 247);</string>
    </property>
    <property name="text">
     <string>Đếm số từ</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonInChuThuong">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>360</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>#9Slide03 Arima Madurai ExtraBo</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background: rgb(220, 214, 247);</string>
    </property>
    <property name="text">
     <string>In chữ thường</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonDemSoKTThuong">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>400</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>#9Slide03 Arima Madurai ExtraBo</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background: rgb(220, 214, 247);</string>
    </property>
    <property name="text">
     <string>Đếm số KT thường</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonInNguyenAmPhuAm">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>400</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>#9Slide03 Arima Madurai ExtraBo</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background: rgb(220, 214, 247);</string>
    </property>
    <property name="text">
     <string>In nguyên âm, phụ âm</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonKetThuc">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>440</y>
      <width>411</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>#9Slide03 Arima Madurai ExtraBo</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background: rgb(220, 214, 247);</string>
    </property>
    <property name="text">
     <string>Kết thúc</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>466</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
